src/components/User.vue

```diff
@@ -1,38 +1,144 @@
+
+    
+    function updateUsername(newUsername: string) {
+      username.value = newUsername;
+    }
+
+    watchEffect(() => {
+      debouncedFetchUserInfo(username.value);
+    });
+
+    return {
+      username,
+      userInfo,
+      error,
+      updateUsername,
+    };
+  },
+});
 </script>
 
 <template>
+    <Search @search="updateUsername" :error="error" />
+    <div v-if="userInfo" class="pt-8 px-6 pb-12 text-xl flex flex-col rounded-2xl border-0 bg-lm_white dark:bg-dm_darkblue shadow-xl">
+      <div class="profile flex flex-row gap-5">
+        <div class="w-16 md:w-28">
+          <img class="rounded-full" :src="userInfo.avatar_url" alt="profile image" />
+        </div>
 
-    <div class="container card text-xl flex flex-col 
-            rounded-2xl border-0 bg-dm_darkblue">
-        
-            <div class="profile flex flex-row">
-                <div class="image">
-                    <img src="" alt="profile image"/>
-                </div>
-                
-                <div class="info flex-col">
-                    <div class="name">Name</div>
-                    <div class="handle">@Handle</div>
-                    <div class="joined">Joined</div>
-                </div>
-            </div>
-            
-            <div class="description">
-                Bio
-            </div>
-            
-            <div class="stats flex flex-row">
-                Repos
-                Followers
-                Following
-            </div>
-    </div>
+        <div class="info flex-col">
+          <div class="name text-base font-bold text-lm_dark dark:text-dm_white">{{ userInfo.name || userInfo.login }}</div>
+          <div class="handle text-xs pb-2 text-primary">@{{ userInfo.login }}</div>
+          <div class="joined text-xs">Joined {{ new Date(userInfo.created_at).toLocaleDateString() }}</div>
+          
+        </div>
+      </div>
 
+      <div class="text-xs pt-6 pb-8">
+        {{ userInfo.bio || 'This profile has no bio' }}
+      </div>
+    
+      <div class="stats flex flex-row justify-evenly rounded-2xl bg-lm_whitegrey dark:bg-dm_black py-5 px-4">
+        <div class="flex flex-col justify-center items-center gap-2">
+          <span class="text-xs">Repos</span>
+          <span class="text-base font-bold text-lm_dark dark:text-dm_white">{{ userInfo.public_repos }}</span>
+        </div>
+        <div class="flex flex-col justify-center items-center gap-2">
+          <span class="text-xs">Followers</span>
+          <span class="text-base font-bold text-lm_dark dark:text-dm_white">{{ userInfo.followers }}</span>
+        </div>
+        <div class="flex flex-col justify-center items-center gap-2">
+          <span class="text-xs">Following</span>
+          <span class="text-base font-bold text-lm_dark dark:text-dm_white">{{ userInfo.following }}</span>
+        </div>
+      </div>
+      
+        <div class="pt-6 flex flex-col gap-4 font-bold">
```

Comment:
I see that on the desktop the layout should be a bit different, will you cover it later?